assets/css/edd-admin.css

```diff
@@ -3265,68 +3272,97 @@ body.download_page_edd-reports {
 	color: #888;
 }
 
-.edd-vertical-sections .section-nav li[aria-selected="true"] a {
-	position: relative;
+.edd-vertical-sections .section-nav .section-title--is-active a {
 	font-weight: bold;
 	color: #555;
 	background-color: #fff;
 	border-right: none;
 	margin-right: -1px;
```

Comment:
I forgot to add this in the PR that implemented `box-shadow`, but we should offset the inner content by another `5px` when active, hovered, or focused, so the icon stays the same distance between the edge and the "border" edge.

Reply:
Can I confess that I prefer the titles/content here not moving around on focus and hover? I know it results in a visual difference between the perceived edge, but I think I prefer the titles to line up regardless of active/focus/hover state. $.02

Reply:
@spencerfinnell I realized that I forgot to tag you in that comment--have made the other changes you requested, but may I leave this one out so that the titles don't jump around on focus/hover?

Reply:
@robincornett I'm fine with that. 👍